src/components/Item/AppItem/getIframeSrc.js

```diff
@@ -0,0 +1,15 @@
+import { FILTER_ORG_UNIT } from '../../../actions/itemFilters.js'
+
+export const getIframeSrc = (appDetails, item, itemFilters) => {
+    let iframeSrc = `${appDetails.launchUrl}?dashboardItemId=${item.id}`
+
+    if (itemFilters[FILTER_ORG_UNIT] && itemFilters[FILTER_ORG_UNIT].length) {
+        const ouIds = itemFilters[FILTER_ORG_UNIT].map(({ id, path }) =>
```

Comment:
@edoardo From what I've seen, all org units regardless if they are LEVEl, GROUP, individual from the hierarchy, or USER_ORG_UNIT* have the `id` property, so it seems unnecessary to parse the `path` at all. Do you remember if there was a reason to parse the path instead of use id?